docs/docs-beta/docs/guides/quality-testing/asset-checks.md

```diff
@@ -1,4 +1,54 @@
 ---
 title: "Test assets with Asset Checks"
 sidebar_position: 10
----
\ No newline at end of file
+---
+
+Asset checks in Dagster provide a way to define and execute different types of data quality checks on your data assets directly in Dagster.
+
+This guide covers the most common use cases for testing assets with asset checks and taking action based on the result.
+
+<details>
+<summary>Prerequisites</summary>
+- Familiarity with [Assets](/concepts/assets)
+</details>
+
+## Testing assets with a single asset check
+
+The example below defines a single asset check on an asset that fails if the `order_id` column of the asset contains a null value.
+
+In this example, the asset check will run after the asset has been materialized, to ensure the quality of its data.
+
+<CodeExample filePath="guides/data-assets/quality-testing/asset-checks/single-asset-check.py" language="python" title="Asset with a single asset check" />
+
+## Testing assets with multiple asset checks
+
+In most cases, checking the data quality of an asset will require multiple checks.
+
+The example below defines two asset checks on an asset using the `multi_asset_check` decorator.
+- one that fails if the `order_id` column of the asset contains a null value.
+- another one that fails if the `item_id` column of the asset contains a null value.
+
+In this example, both asset check will run in a single operation after the asset has been materialized.
```

Comment:
When you say, "a single operation", does that map to an `@op` under the hood?

Reply:
Yes exactly, see below

<img width="1399" alt="Screenshot 2024-08-29 at 1 46 41 PM" src="https://github.com/user-attachments/assets/9961d8e6-df0f-43ef-a145-9527e277f2b8">

Reply:
TIL - thank you.